config/webpack/environment.js

```diff
@@ -1,3 +1,13 @@
 const { environment } = require('@rails/webpacker')
 
+const webpack = require('webpack');
+
+environment.plugins.prepend('Provide', new webpack.ProvidePlugin({
+  $: 'jquery',
+  jQuery: 'jquery',
+  Popper: ['popper.js', 'default']
+}));
+
+environment.config.set('resolve.alias', {jquery: 'jquery/src/jquery'});
```

Comment:
@KevinMulhern Why do we have to require `jquery` from `src` instead of the default `dist` directory?

If possible, please share a link to the documentation which suggests to use the `src` version.

Reply:
No problem @arku, it was in the webpacker docs http://jtdz-solenoids.com/github_/rails/webpacker/blob/master/docs/webpack.md#configuration

Reply:
Got it, thanks!